code/modules/food_and_drinks/food/snacks_vend.dm

```diff
@@ -36,11 +36,11 @@
 	icon_state = "chips"
 	trash = /obj/item/trash/chips
 	bitesize = 1
-	list_reagents = list("nutriment" = 1, "sugar" = 3)
+	list_reagents = list("nutriment" = 1, "sugar" = 3, "sodiumchloride" = 1)
 	junkiness = 20
 	filling_color = "#FFD700"
 	tastes = list("salt" = 1, "crisps" = 1)
-	foodtype = JUNKFOOD | FRIED
+	foodtype = JUNKFOOD | VEGETABLES | FRIED
```

Comment:
JUNKFOOD takes the place of VEGETABLES in this case. That's more for healthy plant food

Reply:
Fair enough